x-pack/plugins/security_solution/server/usage/queries/utils/get_search_for_custom_rules.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  AggregationsAggregationContainer,
+  SearchRequest,
+} from '@elastic/elasticsearch/lib/api/types';
+import { RULE_EXECUTION_LOG_PROVIDER } from '../../../lib/detection_engine/rule_execution_log/event_log/constants';
+
+/**
+ * Given an aggregation of "aggs" this will return a search for rules that are NOT elastic
+ * rules within 24 hours.
+ * @param eventLogIndex The event log index such as ".kibana-event-log-8.2.0*"
+ * @param aggs The aggregation to break things down by
+ * @param elasticRuleIds Array of elastic rule ids to exclude
+ */
+export const getSearchForCustomRules = ({
```

Comment:
Just challenging assumptions here, Frank - but why would we collect this? 
Is it to profile if users are setting up rules properly?

Reply:
I am not for sure if I understand your question. This particular code just breaks things down by custom rules.

Reply:
This was resolved over slack chat, but to summarize we do want to collect this data to analyze it and I think everyone is onboard with adding the data.